geonode/maps/templates/layouts/map_panels.html

```diff
@@ -543,32 +541,109 @@
                     {% endblock map_extra_metadata %}
                 </div>
             </div>
-            <div class="col-xs-12 col-lg-3">
-              <div class="panel panel-default" style="margin-top: 5px">
-                  <div class="panel-heading">{% trans "Responsible Parties" %}</div>
-                  <div class="panel-body">
+            <!-- Contact Roles -->
+            <div class="col-xs-12 col-lg-3" >
+              <div class="panel panel-default" class="collapse" style="margin-top: 5px">
+                <div class="panel-heading">{% trans "Responsible Parties" %}</div>
+                  {% block map_poc %}
+                  <div class="panel-body check-select">
                       <span><label for="{{ map_form.poc|id }}">{{ map_form.poc.label }}</label></span>
                       {{ map_form.poc }}
                   </div>
+                  {% endblock map_poc %}
               </div>
-              <div class="panel panel-default">
-                  <div class="panel-heading">{% trans "Responsible and Permissions" %}</div>
-                  <div class="panel-body">
-                      <div>
+            <div class="panel panel-default">
+                <div class="panel-heading">{% trans "Responsible and Permissions" %}</div>
+                <div class="panel-body">
+                  {% block map_owner %}
+                    <div>
                         <span><label for="{{ map_form.owner|id }}">{{ map_form.owner.label }}</label></span>
-                        <!--<p class="xxs-font-size">(Responsible of the cited resource)</p>-->
                         {{ map_form.owner }}
-                      </div>
-                      <div>
-                          <span><label for="{{ map_form.metadata_author|id }}">{{ map_form.metadata_author.label }}</label></span>
-                          <!--<p class="xxs-font-size">(Author of the metadata)</p>-->
-                          {{ map_form.metadata_author }}
-                      </div>
+                    </div>
+                  {% endblock map_owner %}
+                </div>
+              </div>
+            <a href="#id-more-metadata-panel" data-toggle="collapse">{% trans "toggle more Contact Roles" %}</a>
+            <div class="panel panel-default panel-collapse collapse" collapsed id='id-more-metadata-panel'>

```

Comment:
To reduce the verbosity in the HTML cannot we have a simple loop based on an enum where we can define the names and use it to generate the HTML??